specs/user_spec.rb

```diff
@@ -59,4 +58,40 @@
       end
     end
   end
+
+  describe "User Aggregate Statistics" do
+    before do
+      @user = RideShare::User.new(id: 9, name: "Merl Glover III",
+                                  phone: "1-602-620-2330 x3723", trips: [])
+      trip_1 = RideShare::Trip.new(id: 8, driver: nil, passenger: @user,
+                                 start_time: Time.parse("2016-08-08"),
+                                 end_time: Time.parse("2016-08-09"), cost: 5.25,
+                                 rating: 5)
+      trip_2 = RideShare::Trip.new(id: 8, driver: nil, passenger: @user,
+                                  start_time: Time.parse("2016-08-07"),
+                                  end_time: Time.parse("2016-08-10"), cost: 10.84,
+                                  rating: 5)
+      trip_3 = RideShare::Trip.new(id: 8, driver: nil, passenger: @user,
+                                  start_time: Time.parse("2016-08-01"),
+                                  end_time: Time.parse("2016-08-03"), cost: 35.20,
+                                  rating: 5)
```

Comment:
Looks like you didn't come back to test that your stuff from wave 1 works with your incomplete trips from Wave 3!